### PR TITLE
Discovery 5.0.2 update info

### PR DESCRIPTION
**Pull request by lzap**
Release pending.

### CODE REVIEW

File: security.md [hunk 1]
```diff
@@ -54,6 +55,17 @@ All security advisories made for Foreman are listed below with their correspondi
 
 ### Disclosure details
 
+#### <a id="2016-discovery-settings"></a>Ruby code execution via Discovery settings
+
+It is possible to run arbitrary Ruby code entering it on the About - Settings
```
Comment (domcleal) [inline security.md]: "About" should be "Administer"

File: security.md [hunk 2]
```diff
@@ -54,6 +55,17 @@ All security advisories made for Foreman are listed below with their correspondi
...
+- Discovery and then visiting a discovered host detail page where it gets
```
Comment (domcleal) [inline security.md]: Don't start this line with a hyphen, it renders as a list.

File: security.md [hunk 3]
```diff
@@ -54,6 +55,17 @@ All security advisories made for Foreman are listed below with their correspondi
...
+rendered.
+
+* Affects Foreman Discovery 4.2.0 to 5.0.1 (gem releases only, nightly builds)
```
Comment (domcleal) [inline security.md]: also 1.11 RC builds

**Comment by lzap** [comment]
Thanks, that manual formatting is unfortunate. All fixed!

**Comment by domcleal** [comment]
The commit is unchanged, please push your changes.

**Comment by lzap** [comment]
Damn, right. Done.

**Comment by domcleal** [comment]
Merged, thanks @lzap.

**Pull request merged**